treeherder/model/models.py

```diff
@@ -188,12 +189,17 @@ class Commit(models.Model):
 
     class Meta:
         db_table = "commit"
-        unique_together = ("push", "revision")
+        unique_together = ("push", "revision"),
+        indexes = [
+            GinIndex(fields=['revision', 'author', 'comments']),
```

Comment:
Lookup the doc in https://docs.djangoproject.com/en/5.1/ref/contrib/postgres/search/#performance, we want a GinIndex on a SearchVector.
Don't forget to add a `config="english"` also on the index, this seems important to make sure it's properly used later.

Reply:
I think the GinIndex will need to be put in the Push model, because that's where it will be used (I'm not so sure, we may need to ask somebody who knows more than myself in django x postgresql)

Reply:
Okay thanks!